gno.land/pkg/gnoclient/signer.go

```diff
@@ -124,7 +124,7 @@ var _ Signer = (*SignerFromKeybase)(nil)
 // This can be useful in scenarios where storing private keys in the filesystem isn't feasible.
 //
 // Warning: Using keys.NewKeyBaseFromDir is recommended where possible, as it is more secure.
-func SignerFromBip39(mnemonic string, passphrase string, account uint32, index uint32) (Signer, error) {
+func SignerFromBip39(chainID string, mnemonic string, passphrase string, account uint32, index uint32) (Signer, error) {
```

Comment:
```suggestion
func SignerFromBip39(mnemonic, passphrase , chainID string, account, index uint32) (Signer, error) {
```

I suggest reordering like this.